nibabel/nicom/ascconv.py

```diff
@@ -89,7 +89,10 @@ def assign2atoms(assign_ast, default_class=int):
             target = target.value
             prev_target_type = OrderedDict
         elif isinstance(target, ast.Subscript):
-            index = target.slice.value.n
+            if isinstance(target.slice, ast.Constant):  # PY39
+                index = target.slice.n
+            else:  # PY38
+                index = target.slice.value.n
```

Comment:
cc @moloney @matthew-brett Apparently there's a change in `ast` behavior in Python 3.9. IDK if we need to be checking other sections for more subtle bugs?

Reply:
Glad you tracked that one down.  Yes we should probably comb ...  but it will be a hard job.